build-system/tasks/e2e/functional-test-types.js

```diff
@@ -0,0 +1,105 @@
+/**
```

Comment:
Can we get rid of "functional test" altogether and call this `e2e-types.js` or just `types.js`?